src/Polly.Net35/CircuitBreaker/SuccessRatioCircuitBreakerState.cs

```diff
@@ -66,6 +77,18 @@ public void TryBreak(Exception ex)
             }
         }
 
+        private double Decay(double counter, DateTime lastUpdate)
+        {
+            var now = DateTime.UtcNow;
+
+            if (counter > 0.00001)

```

Comment:
it's really because resharper shows warning about floating point equality comparison if you try
counter == 0.0
and we want to stop decaying once we essentially reach zero.